blogs/ecs-canary-deployments-pipeline/setup/scripts/export_environment_variables.sh

```diff
@@ -4,7 +4,7 @@ set -e
 AWS_REGION=us-west-2
 ENVIRONMENT_NAME=ecs-blogpost
 NAMESPACE=yelb.local
-ENVOY_IMAGE=840364872350.dkr.ecr.${AWS_REGION}.amazonaws.com/aws-appmesh-envoy:v1.15.1.0-prod
```

Comment:
We can, because envoy's configuration is fully EMS-hosted, so I use colormap example test all can, 1.17 is the most changable version, then I change to the latest stable version 1.24 it

Reply:
Gotcha. Thanks Sunil